mail_activity_plan/models/mail_activity_mixin.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2024 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from lxml import etree
+
+from odoo import _, api, fields, models
+
+from odoo.addons.base.models import ir_ui_view
+
+
+class MailActivityMixin(models.AbstractModel):
+    _inherit = "mail.activity.mixin"
+
+    has_activity_plans = fields.Boolean(
```

Comment:
Avoid plurals as possible:

```suggestion
    has_activity_planning = fields.Boolean(
```